modules/nw-multus-macvlan-object.adoc

```diff
@@ -43,6 +43,11 @@ plug-in:
 
 |====
 
+[NOTE]
+====
+If you specify the `master` key for the plug-in configuration, do not specify the same physical network interface associated with your primary network plug-in to avoid possible conflicts.
```

Comment:
```suggestion
If you specify the `master` key for the plug-in configuration, use a different physical network interface than the one that is associated with your primary network plug-in to avoid possible conflicts.
```